Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -152,6 +152,7 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   ((result (*)(id, SEL, ...)) impl)(class, selector, ##__VA_ARGS__); \
 })
 
+#if TARGET_OS_IOS
 + (void)makeKeyAndVisible {
   if (@available(iOS 13.0, *)) {
```

Comment:
```
  if (@available(iOS 13.0, tvOS 13.0, *)) {
```
This doesn't work?

Reply:
I agree this fix is better.  I was a bit confused by tvOS simulators versions.
I'll update this PR.